AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -344,7 +344,7 @@ - (void)setNeedsLayout
     // The node is loaded but we're not on main.
     // We will call [self __setNeedsLayout] when we apply
     // the pending state. We need to call it on main if the node is loaded
-    // to support implicit hierarchy management.
+    // to support automatic hierarchy management.
```

Comment:
Let's be extremely consistent here and use `to support automatic subnode management`